Admin/Extension/PublishTimePeriodExtension.php

```diff
@@ -45,15 +51,27 @@ public function configureFormFields(FormMapper $formMapper)
             'required' => false,
         );
 
-        $formMapper->with($this->formGroup, array(
-            'translation_domain' => 'CmfCoreBundle',
-            ))
-            ->add('publish_start_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_start_date',
-            ))
-            ->add('publish_end_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_end_date',
+        if ($formMapper->hasOpenTab()) {
```

Comment:
as it's part of `FormMapper` (which we typehint in the method), I'm not sure why we should do that